include/envoy/http/codec.h

```diff
@@ -146,6 +146,19 @@ class ConnectionCallbacks {
    */
   virtual void onGoAway() PURE;
 };
+/**
+ * HTTP/1.* Codec settings
+ */
+
+struct Http1Settings {
+  // Enable codec to handle CONNECT requests. This will enable forward/explicit proxy support on TLS
+  // connections
+  // TODO(mattwoodyard) - Implement CONNECT support.
+  bool allow_connect_{false};
```

Comment:
Can we just del this for now? Any reason to have it in here? CONNECT needs a ton of thinking in terms of how it might be implemented.